### PR TITLE
Refactor MyBooksViewController 

### PR DESCRIPTION
**Pull request by mauricecarrier7**
**What's this do?**
- Refactors MyBooksViewController to SwiftUI
- Refactors book cells to SwiftUI

**Why are we doing this? (w/ Notion link if applicable)**
https://www.notion.so/lyrasis/Audiobook-playing-twice-4abc3e87e31743ab81a65a58092cf06a

**How should this be tested? / Do these changes have associated tests?**
- A full regression of MyBooks view and all related functionality should be performed

**Dependencies for merging? Releasing to production?**
N/A

**Does this include changes that require a new Palace build for QA?**
Yes

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
@mauricecarrier7 

<details>

https://user-images.githubusercontent.com/6921353/221278169-d43c458b-9bf3-45c8-9ef3-3fb7fcbab160.MP4

https://user-images.githubusercontent.com/6921353/221297626-1deac25e-b8e4-4310-ae0d-051d44448ad5.MP4

</details>

### CODE REVIEW

**Review by vladimirfedorov — CHANGES_REQUESTED** [review]

File: Palace/MyBooks/MyBooks/BookCell/BookCellModel.swift [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    }
+  }
+
+  func didSelectRead() {
+    isLoading = true
+    self.buttonDelegate?.didSelectRead(for: book)
+    TPPRootTabBarController.shared().dismiss(animated: true)
+  }
+
+  func didSelectReturn() {
```
[the remark this answers is not in the document]
Comment (mauricecarrier7) [reply Palace/MyBooks/MyBooks/BookCell/BookCellModel.swift:153]: Caused by navigation stack issues, resolved

File: Palace/MyBooks/MyBooks/MyBooksViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  }
+  @Published var accountURL: URL?
+  
+  var activeFacetSort: Facet {
+    didSet {
+      sortData()
+    }
+  }
+
+  let facetViewModel: FacetViewModel = FacetViewModel(
```
[the remark this answers is not in the document]
Comment (mauricecarrier7) [reply Palace/MyBooks/MyBooks/MyBooksViewModel.swift:38]: Navigation issue resolved

File: Palace/MyBooks/MyBooks/MyBooksViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    }
+
+    if TPPUserAccount.sharedAccount().needsAuth && !TPPUserAccount.sharedAccount().hasCredentials() {
+      TPPAccountSignInViewController.requestCredentials(completion: nil)
+    } else {
+      TPPBookRegistry.shared.sync()
+    }
+  }
+  
+  func refresh() {
```
[the remark this answers is not in the document]
Comment (mauricecarrier7) [reply Palace/MyBooks/MyBooks/MyBooksViewModel.swift:130]: updated

**Review by vladimirfedorov — APPROVED** [review]
Looks good to me

**Pull request merged**